content/en/metrics/custom_metrics/historical_metrics.md

```diff
@@ -0,0 +1,453 @@
+---
+
+## Overview
+
+Historical Metrics Ingestion allows you to collect metric points with outdated timestamps. Timestamps that are older than one hour from the time of submission, but no older than your total metric retention period (which defaults to 15 months for all metrics).
+
+With Historical Metrics Ingestion, you can monitor a variety of new use cases with your metrics, such as outage recovery, overwriting invalid metrics, and managing IoT delays.
+
+## What is Historical Metric Ingestion?
+
+{{< img src="metrics/custom_metrics/historical_metrics/diagram_historical-metrics-ingestion_intro.png" alt="Diagram showing the ingestion flow for Historical Metrics" >}}
+
+Datadog classifies *historical metrics* as metric points with timestamps that are older than an hour relative to the time of submission. 
+
+For example, you emit a metric point at 1:00 PM EST, and the timestamp on that point reads 10:00 AM EST. This metric point is classified as a historical metric because it is delayed by 3 hours relative to the time of submission.
+
+Metric points that are resent with the same timestamp and tag-value combination within Datadog are replaced with a *"last point wins"* ingestion rule. If you send a metric with a value of 5, and resend that metric with a value of 10, both with the same timestamp, the 10 replaces the 5 as the value of the metric.
+
+You can start ingesting historical metrics by configuring Historical Metrics Ingestion on the [Metrics Summary Page][1] for *counts, rates, and gauges*.
+
+## Configuration
+
+To enable the ingestion of historical metrics for a specific metric:
+1. Navigate to the [Metrics Summary Page][1].
+1. Click the name of the metric you want to enable Historical Metrics Ingestion for. 
+1. On the metric side panel that opens up, under the **Advanced** section find the option for *Historical Data*. 
+1. Click **Configure**.
+1. Select **Enable historical metrics** and press **Save**.
+
+{{< img src="metrics/custom_metrics/historical_metrics/enable_historical_metrics.png" alt="Metrics Summary page showing the Historical Metrics facets panel and the Advanced section of an open Metric detail panel with the Enable historical metrics option selected" style="width:100%;" >}}
+
+### Bulk configuration
+
+To optimize your historical metrics enablement, take advantage of the Bulk Historical Metric Enablement feature. You can enable Historical Metrics Ingestion for multiple metrics at once, rather than having to configure each one individually.
+
+1. Navigate to the Navigate to the [Metrics Summary Page][1] and click the **Configured Metrics** dropdown.
+1. Select **Enable historical metrics**.
+1. Configure all metrics that match that namespace to enable historical metrics ingestion.
+
+{{< img src="metrics/custom_metrics/historical_metrics/enable_bulk_historical_metrics.mp4" alt="Walkthrough of bulk enabling historic metric ingestion" video=true >}}
```

Comment:
Can we have a video of bulk config with a metric namespace prefix that matches more than just one metric name?